src/foam/u2/PropertyBorder.js

```diff
@@ -94,7 +94,7 @@ foam.CLASS({
           if ( prop.validateObj && prop.internalValidateObj ) {
             slot = foam.core.ExpressionSlot.create({
               args: [ this.data.slot(prop.validateObj), this.data.slot(prop.internalValidateObj) ],
-              code: function (e1, e2) { return e1 ? e1 + ' ' + ( e2 || '' ) : e2; }
+              code: function (e1, e2) { return e2 ? e2 : e1; }
```

Comment:
This was a feature.

Reply:
Example:     { class: 'Int',         name: 'minMaxOdd', min: 1, max: 10, validateObj: function(minMaxOdd) { if ( minMaxOdd % 2 == 0 ) return 'Number should be odd.'; } },